README.md

```diff
@@ -1,2 +1,124 @@
+
+#### Annotations Table
+
+create_Syn_table_from_Syn_schemas.py will generate a Synapse table of all terms found in a set of metadata templates. The set is determined by consortium using the config file (config/schemas.yml). There are options to create a new table or overwrite an existing table.
+
+Parameters:
+
+- `--config_file <YAML file>`: Path to the config file.
+- `--consortium <consortium>`: The consortium to create the table for; will only gather terms in registered templates for that consortium.
+- `new_table`  OR `overwrite_table`: Choose to create a new table or overwrite an existing table.
+  - For `new_table`
+    - `--parent_synapse_id <synID>`: synID for project that table should be created in.
+    - `--synapse_table_name <table name>`: Name for the new table.
+  - For`overwrite_table`
+    - `--table_synapse_id <synID>`: synID of table to overwrite.
+
+#### Generate Metadata Template
+
+create_template_from_Syn_schema.py will generate either a .csv or .xlsx metadata template based on a registered metadata schema. Note that this script will not upload the file to Synapse automatically.
+
+Parameters:
+
+- `<registered schema id>`: id of the registered metadata schema, including organization (e.g. sysbio.metadataTemplates_assay.STARRSeq).
+- `<output file path>`: Full path to the where the template file should be output.
+- `csv`  OR `excel`: Choose output as .csv (csv) or .xlsx (excel). A csv file will only include a header with the metadata keys. An excel file will include three sheets: one with a header of metadata keys, one with a dictionary of key descriptions, and one with a dictionary of allowed values plus their descriptions.
+
+### Docker
+
+This repo contains a Dockerfile that can be used to build a docker image locally. Alternatively, the docker image is on Docker Hub under [aryllen/sysbioDCCjsonschemas](https://hub.docker.com/repository/docker/aryllen/sysbiodccjsonschemas).
+
+##### Build Image Locally
+
+If you'd like to build the docker image locally, clone this repo and open a terminal in the sysbioDCCjsonschemas folder. Then build image and run interactively.
+
+```bash
+git clone https://github.com/Sage-Bionetworks/sysbioDCCjsonschemas.git
+cd sysbioDCCjsonschemas
+docker build -t sysbiodccjsonschemas .
+docker run --rm -it sysbiodccjsonschemas
+```
+
+##### Pull Existing Image
+
+If you'd like to use an existing image, then pull the docker image from Docker Hub. Below assumes pulling the latest version of the image. To use a different version, replace `latest` with the [desired tag](https://hub.docker.com/repository/docker/aryllen/sysbiodccjsonschemas/tags?page=1&ordering=last_updated). The container can be run interactively once the image is pulled.
+
+```
+docker pull aryllen/sysbiodccjsonschemas:latest
+docker run --rm -it aryllen/sysbiodccjsonschemas:latest
+```
+
```

Comment:
```suggestion
Because the docker image is not currently auto-deployed, it may be out of date with the repo. It is recommend to build the image locally if you are: 

- adding a new key to an existing template
- adding an entirely new template
- generating the annotation table when a new template has been added to the config

```